projects/GKCore/GKCore/LangMan.cs

```diff
@@ -991,7 +991,7 @@ public static class LangMan
             /* 099 */ "Cancel",
             /* 100 */ "Close",
             /* 101 */ "Select",
-            /* 102 */ "Append",
+            /* 102 */ "Add new",

```

Comment:
This is a button in record selection dialog. "Append" does not correctly describe the action that will be performed by the button.

Reply:
👍